extension/ui/components/assistants/state/messageReducer.ts

```diff
@@ -157,6 +157,7 @@ export function messageReducer(
     case "reasoning_thinking":
     case "reasoning_tokens":
     case "retrieval_params":
+    case "search_labels_params":
```

Comment:
You had an error when building the ext locally? (We need to clean that up this is due to the migration of old actions to MCP - we need to update the SDK first)

Reply:
Indeed I had an error when building locally. We discussed IRL, I'll leave it as is it for now.